zulipterminal/config/keys.py

```diff
@@ -109,6 +109,11 @@
         'help_text': 'Autocomplete @mentions and #stream_names',
         'key_category': 'msg_compose',
     }),
+    ('AUTOCOMPLETE_REVERSE', {
+        'keys': {'ctrl z'},
+        'help_text': 'Go back to previous autocomplete suggestion',
+        'key_category': 'msg_compose',
+    }),
```

Comment:
I don't know if `ctrl z` is the best option here. Open to suggestions for alternatives. I wanted to use `ctrl shift f` here, but for some reason it doesn't seem to work :/

Reply:
For me, `ctrl shift f` popped up a find box in my terminal :)

Reply:
We may be able to disable this, if that turns out to be a good idea